atom/common/platform_util_win.cc

```diff
@@ -231,6 +232,7 @@ HRESULT DeleteFileProgressSink::ResumeTimer() {
 namespace platform_util {
 
 bool ShowItemInFolder(const base::FilePath& full_path) {
+  base::ScopedAllowBlockingForTesting allow_blocking;
```

Comment:
the `ForTesting` here means that this code is supposed to only be invoked in tests. Either make this non-blocking, find a "blessed" (non-testonly) way to allow blocking, or add a comment here as to why we can't do either of those things.

Reply:
There is a class that's exactly like this, but only allows blocking for friends, and I didn't want to make a patch just for that one line if we have a version that works, but if that sounds more reasonable to you, I'll do that. I guess patches are a bit less of a concern now that we don't have to wait for libcc?

Reply:
What is it in this PR that causes the additional need for this blocking-permitter?

Reply:
I came across this bug while using the [printing sample app](https://github.com/hokein/electron-sample-apps/tree/master/printing), it looks like the calls to `OpenFolderViaShell` and such are now marked as only callable from a thread that can block. We could break this out to a separate PR.